pkgs/development/cuda-modules/generic-builders/manifest.nix

```diff
@@ -170,13 +170,15 @@ backendStdenv.mkDerivation (
       ''
       # Handle the existence of libPath, which requires us to re-arrange the lib directory
       + strings.optionalString (libPath != null) ''
-        if [[ ! -d "${libPath}" ]] ; then
-          echo "${finalAttrs.pname}: ${libPath} does not exist, only found:" >&2
-          find "$(dirname ${libPath})"/ -maxdepth 1 >&2
+        full_lib_path="lib/${libPath}"
```

Comment:
I've also been wondering lately if it's better to pass this kind of values as environment variables instead of rendering them in the bash script, but that's not relevant rn I guess